src/background/stores/details-view-store.ts

```diff
@@ -39,9 +43,6 @@ export class DetailsViewStore extends BaseStoreImpl<DetailsViewStoreData> {
     }
 
     protected addActionListeners(): void {
-        this.previewFeaturesActions.openPreviewFeatures.addListener(() =>
```

Comment:
It looks like both the only listener and the only invoker of `PreviewFeaturesActions.openPreviewFeatures` are removed in this PR; should the action also be removed from `preview-features-actions`?

Reply:
Good catch! Updated.